extensions/Worker.Extensions.EventGrid/src/TypeConverters/EventGridCloudEventConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json;
+using Azure.Messaging;
+using Microsoft.Azure.Functions.Worker.Converters;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.EventGrid.TypeConverters
+{
+    /// <summary>
+    /// Converter to bind to <see cref="CloudEvent" /> or <see cref="CloudEvent[]" /> type parameters.
+    /// </summary>
+    [SupportedTargetType(typeof(CloudEvent))]
+    [SupportedTargetType(typeof(CloudEvent[]))]
+    internal class EventGridCloudEventConverter: EventGridConverterBase
+    {
+        protected override ConversionResult ConvertCore(Type targetType, string json)
+        {
+            if (targetType != typeof(CloudEvent) && targetType != typeof(CloudEvent[]))
```

Comment:
Is this check needed anymore since 14 and 15 causes the calling code to do same?